logprep/processor/base/rule.py

```diff
@@ -255,21 +255,23 @@ def lucene_filter(self):
     # pylint: enable=C0111
 
     @classmethod
-    def create_rules_from_target(cls, path: str) -> list:
+    def create_rules_from_target(cls, rule_target: str) -> list:
```

Comment:
Depending on how you decide to implement my previous remark you could either rename this `rule_target` to something similar to `rule_description` or you just have here a list of rule dicts anways such that this whole method could be simplified.

Reply:
we will challenge that later